components/board/digital_interrupts_test.go

```diff
@@ -25,25 +25,30 @@ func TestBasicDigitalInterrupt1(t *testing.T) {
 	intVal, err := i.Value(context.Background(), nil)
```

Comment:
would add a test on using the digital interrupt tick to measure a time delta. Especially testing that timeB-timeA give the right result even when timeB < timeA (so after a rollover)